### PR TITLE
Updated helm chart with new defaults so the documentation works.

### PR DESCRIPTION
**Pull request by trevorbox**

### CODE REVIEW

File: config/helmchart/values.yaml.tpl [hunk 1]
```diff
@@ -16,19 +16,19 @@ fullnameOverride: ""
...
 serviceAccount:
   # Specifies whether a service account should be created
-  create: true
+  create: false
   # Annotations to add to the service account
   annotations: {}
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
-  name: ""
+  name: "default"
```
Comment (raffaelespazzoli) [inline config/helmchart/values.yaml.tpl:24]: why this change? and the one above?

File: config/helmchart/values.yaml.tpl [hunk 2]
```diff
@@ -16,19 +16,19 @@ fullnameOverride: ""
...
 
 podAnnotations: {}
 
 resources:
-  limits:
-    cpu: 100m
-    memory: 30Mi
+  #limits:
```
Comment (raffaelespazzoli) [inline config/helmchart/values.yaml.tpl:29]: please just delete these.

**Review by raffaelespazzoli — CHANGES_REQUESTED** [review]

File: config/helmchart/values.yaml.tpl (hunk 1 above)
Comment (trevorbox) [reply config/helmchart/values.yaml.tpl:24]: It will only work if using the default serviceaccount right now (rolebindings from kustomization use default).

File: config/helmchart/values.yaml.tpl (hunk 2 above)
Comment (trevorbox) [reply config/helmchart/values.yaml.tpl]: removed.

**Comment by trevorbox** [comment]
new version from https://github.com/redhat-cop/proactive-node-scaling-operator/pull/7
